RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -165,7 +196,11 @@ struct SubscriptionDetailsView: View {
                     .foregroundColor(Color.gray)
                     .padding([.horizontal, .bottom])
             }
-        }
+        }.padding()
+            .padding(.horizontal)
```

Comment:
Hmm this looks weird to me. Are we adding `padding` twice?

Reply:
We are adding a bit more padding horizontally than we do vertically. (The first `padding()` adds padding to all sides.) Afaik this is common in layouts to have a bit more breathing room on the sides. 

The reason this is not done in 1 go is because I don't know what the default padding is that's added by `padding()`.  The docs just say "a default amount of padding". 🤷